rootdir/Android.mk

```diff
@@ -100,27 +100,27 @@ LOCAL_SRC_FILES := vendor/etc/init/cdsprpcd.rc
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR)/etc/init
 include $(BUILD_PREBUILT)
-endif # TARGET_BOARD_PLATFORM in (sdm845 sm6125 sm6350 sm8150)
+endif
 
-ifneq ($(filter sdm660 msm8998 sdm845 sm8150,$(TARGET_BOARD_PLATFORM)),)
+ifeq ($(TARGET_NEEDS_SSCRPCD), true)
 include $(CLEAR_VARS)
 LOCAL_MODULE := sscrpcd.rc
 LOCAL_MODULE_CLASS := ETC
 LOCAL_SRC_FILES := vendor/etc/init/sscrpcd.rc
 LOCAL_MODULE_TAGS := optional
 LOCAL_MODULE_PATH := $(TARGET_OUT_VENDOR)/etc/init
 include $(BUILD_PREBUILT)
-endif # TARGET_BOARD_PLATFORM in (sdm660 msm8998 sdm845 sm8150)
+endif
 
-ifneq ($(filter sm6125 sm6350 ,$(TARGET_BOARD_PLATFORM)),)
+ifeq ($(TARGET_NEEDS_HVDCP_OPTI), true)
```

Comment:
Can we make this one more explanatory? `TARGET_NEEDS_USERSPACE_CHARGER_CURVE_GARBAGE_COMPUTERATOR`?

Reply:
No we cant.

Reply:
If I want it, I can :P